cmk/special_agents/agent_cisco_meraki.py

```diff
@@ -220,7 +317,11 @@ def query(self) -> Iterator[Section]:
             )
 
         if _SEC_NAME_DEVICE_STATUSES in self.config.section_names:
-            for device_status in self._get_device_statuses():
+            for device_status in MerakiGetOrganizationDevicesStatuses(
```

Comment:
The initialization of this (and the following) classes can be moved to the init function (composite).